packages/jest-preset/mocks/hops.js

```diff
@@ -1,5 +1,5 @@
 /* eslint react/prop-types: 0 */
-const hops = require('hops/lib/runtime');
+const hops = require('untool').internal.runtime;
```

Comment:
Is this not somewhat breaking? After all, `hops/lib/runtime` exports quite a bit more than `untool`'s runtime...

Reply:
Yes, you're right. I've reverted this change.